docs/reference/administration_reference.md

```diff
@@ -0,0 +1,74 @@
+# Administration reference guide
+
+## Overview
+
+This document provides a reference guide for all the attributes available for `invenio-administration` views.
+
+## List View
+
+### Attributes
+
+- **name**                                 (*View class name*)                           - view's name used by Flask to create routing
+- **resource_config**          *required*                                                - used to retrieve the resource (refers to extension attribute - check ext.py)
+- **search_request_headers**               (*{ "Accept" :  "application/json" }*)        - used to request the resource
+- **title**                    *required*                                                - displays a title in the list view page.
+- **category**                 *required*                                                - places the menu entry under a category in the menu bar.
+- **pid_path**                             (*"pid"*)                                     - used for accessing the unique identifier of a resource 
+- **icon**                                                                               - displays an icon next to the menu's entry.
+- **template**                             (*"invenio_administration/search.html"*)      - template to be rendered for the view. By default, it uses a predefined one. Can be overridden with a custom template.
+- **display_search**                       (*True*)                                      - displays a searchbar in the list view.
+- **display_delete**                       (*False*)                                     - displays a delete button for each resource.
+- **display_edit**                         (*False*)                                     - displays an edit button for each resource.
+- **item_field_list**          *required*                                                - list of fields to be rendered in the list view's table. Each field  corresponds to a column.
+- **search_config_name**                   (*view's name*)                               - search app configuration name
+- **search_facets_config_name**                                                          - displays facets in search table
```

Comment:
maybe too late, but above flags to displaye are called "display_", do they serve the same purpose? just to consolidate naming.

Reply:
I am not sure I understand the question. They have similar purpose, but how would you consolidate the naming ?

Reply:
sorry, badly phrased. If they are all "flags" to turn on/off displaying something, why the previous starts by `display_` but this ones by `search_`. 

I do not mean to change implementation so... this can be ignored.

Reply:
ah, the description is written in misleading way, I will fix it.
It is not a display flag, its a pointer to facet config object.

Reply:
Ahhh thanks, I thought it was a on/off flag. Thanks! resolving...